### ISSUE TITLE
Install from URL not working

### ISSUE THREAD

**Issue by Napoleon-BlownApart** [joomla/joomla-cms#13944]
re: 3.7 Staging as of 2017-01-27  

The _Install from URL_ has issues and does not allow access to the URL entered (at least some).  See attached/referred image:  [Error when installing from URL](https://www.abms.net.au/public/J37_Staging_Install_From_URL_not-working.png).  The URL in the image is easily resolved by the browser in which Joomla is being accessed.  (I use the _windows.int_ domain on my lan and have it defined in the hosts file on each machine.)

So, instead of installing from the URL, I downloaded from the URL first, then used _Upload Package File_.

**Comment by infograf768** [comment]
I guess "(at least some)" is the key here...

**Comment by tonypartridge** [comment]
This isn't usually a Joomla! Problem but a server side constraint either resolving the URL or invalid ssl certificates.

**Comment by Napoleon-BlownApart** [comment]
It happened, I found it odd that it didn't work, so I posted.
Nothing wrong with the URL entered.  I have no problems resolving any of my other local domains.   I cache software I use locally to reduce bandwith waste.

I tried again with a fresh install of the same commit and the same problem occurred again.  When I copy the url into my browser, it downloads the zip.   The error is repeatable.

With all the character filtering and other security checks that take place in the code, htmlspecialcharacters() ect, there could be something there.  If you've used a hosts file before, you'll understand that it's not a server side issue, DNS, or anything else like that.
